analyzer/codechecker_analyzer/cmd/checkers.py

```diff
@@ -270,26 +270,6 @@ def __uglify(text: str) -> str:
     return text.lower().replace(' ', '_')
 
 
-def guideline_rules_for_checker(
```

Comment:
No longer needed? This seems to be gone for good.

Reply:
Please, double check it, but I haven't found it being used.

Reply:
This function was never really used. It was an alternative implementation which was commented out from the source code: https://github.com/Ericsson/codechecker/blob/1ed457ea8a811ac1b66860d76c46936471cf8e86/analyzer/codechecker_analyzer/cmd/checkers.py#L374